DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/ToHost/Raw/Geometry/ArcConverterToHost.cs

```diff
@@ -0,0 +1,57 @@
+using Speckle.Converters.Common.Objects;
+using Speckle.Converters.RevitShared.Services;
+using Speckle.Core.Common;
+
+namespace Speckle.Converters.RevitShared.ToHost.Raw.Geometry;
+
+public class ArcConverterToHost : ITypedConverter<SOG.Arc, DB.Arc>
+{
+  private readonly ScalingServiceToHost _scalingService;
+  private readonly ITypedConverter<SOG.Point, DB.XYZ> _pointToXyzConverter;
+  private readonly ITypedConverter<SOG.Plane, DB.Plane> _planeConverter;
+
+  public ArcConverterToHost(
+    ITypedConverter<SOG.Point, DB.XYZ> pointToXyzConverter,
+    ScalingServiceToHost scalingService,
+    ITypedConverter<SOG.Plane, DB.Plane> planeConverter
+  )
+  {
+    _pointToXyzConverter = pointToXyzConverter;
+    _scalingService = scalingService;
+    _planeConverter = planeConverter;
+  }
+
+  public DB.Arc Convert(SOG.Arc target)
+  {
+    double startAngle,
+      endAngle;
+    if (target.startAngle > target.endAngle)
+    {
+      startAngle = (double)target.endAngle;
+      endAngle = (double)target.startAngle;
+    }
+    else
+    {
+      startAngle = (double)target.startAngle.NotNull();
+      endAngle = (double)target.endAngle.NotNull();
+    }
+    var plane = _planeConverter.Convert(target.plane);
```

Comment:
whitespace, this is just me asking for us to ensure we have a line space between the start and end of any scope